tests/transports-integrations/tests/utils/common.py

```
@@ -635,6 +650,186 @@ def assert_error_propagation(error_response: Any, integration: str):
+        ), "Google streaming chunk should have at least one candidate"
+
+        candidate = chunk.candidates[0]
+        assert hasattr(candidate, "content"), "Google candidate should have content"
+
+        if is_final:
+            assert hasattr(
+                candidate, "finish_reason"
+            ), "Final chunk should have finish_reason"
+
+
+def collect_streaming_content(
```

Ruff (0.12.2)

`collect_streaming_content` is too complex (19 > 10)

(C901)